src/pkgx.dev/TeaProtocol.tsx

```diff
@@ -1,237 +1,217 @@
+  },
+  {
+    title: "teaRANK: The PageRank for Open Source",
+    description:
+      "Open source powers everything, but which packages power open source? teaRANK, tea’s novel global dependency tree, ranks software based on real usage and impact—just like PageRank did for the web. Integrated into Chai, our open-source intelligence engine, teaRANK ensures that the most critical packages receive the support they deserve."
+  },
+  {
+    title: "PKGX & teaBase: Developer Experience, Reinvented",
+    description:
+      "Developers deserve magic, not headaches. PKGX, our next-gen package manager, makes installing, updating, and securing dependencies effortless. Meanwhile, teaBase serves as the on-chain registry, ensuring that every piece of open source has a permanent, verifiable home—with incentives baked in."
+  },
+  {
+    title: "Open Source Should Be Rewarding. Now It Is.",
+    description:
+      "For too long, open source has been powered by passion, not paychecks. tea is changing the economics of open source, ensuring that every maintainer, contributor, and developer gets their fair share. Whether through staking, governance, or direct rewards, tea.xyz is about sustaining open source for the long run—so you can keep building, innovating, and, of course, sipping the tea."
+  }
 ];
 
-interface TeaStats {
-  totalBlocks: string;
-  dailyTransactions: string;
-  totalTransactions: string;
-  walletAddresses: string;
-}
-
 const TeaProtocol = () => {
-  const [stats, setStats] = React.useState<TeaStats | null>(null);
+  const [stats, setStats] = React.useState(fallbackStats);
 
   React.useEffect(() => {
-    const fetchTeaStats = async () => {
-      // lambda proxy to bypass CORS
-      const response = await fetch(`https://yo2fkzmf2rh33u2b3bi3xhtqyi0toyqz.lambda-url.us-east-1.on.aws/?url=/stats`);
-      const data = await response.json();
-
-      setStats({
-        totalBlocks: parseInt(data.total_blocks).toLocaleString(),
-        dailyTransactions: `${(parseInt(data.transactions_today) / 1000000).toFixed(2)}M`,
-        totalTransactions: parseInt(data.total_transactions).toLocaleString(),
-        walletAddresses: parseInt(data.total_addresses).toLocaleString()
-      });
+    const fetchStats = async () => {
+      try {
+        const format = (val: string | number) => parseInt(val.toString()).toLocaleString();
+
+        const fetchJSON = async (url: string) => {
+          const res = await fetch(url);
+          if (!res.ok) throw new Error("Request failed");
+          return res.json();
+        };
+
+        const [blocksRes, txsRes, walletsRes] = await Promise.all([
+          fetchJSON("https://assam.tea.xyz/api?module=proxy&action=eth_blockNumber"),
+          fetchJSON("https://assam.tea.xyz/api?module=stats&action=txsCount"),
+          fetchJSON("https://assam.tea.xyz/api?module=stats&action=addressCount")
```

Comment:
is this supposed to be assam or sepolia?

Reply:
@jhheider Timothy requested Assam numbers